examples/wasm-jupyterlite/pixi.toml

```
@@ -0,0 +1,44 @@
+[project]
+name = "wasm"
+version = "0.1.0"
+description = "Setup a pixi environment for building a jupyterlite app with WASM"
+authors = ["Wolf Vollprecht <wolf@prefix.dev>"]
+platforms = ["emscripten-wasm32", "osx-arm64", "osx-64", "linux-64", "win-64"]
+channels = ["conda-forge"]
+
+[feature.wasm]
+# add the emscripten-forge channel to the list of channels
+channels = ["https://repo.mamba.pm/emscripten-forge"]
+platforms = ["emscripten-wasm32"]
+
+[feature.wasm.dependencies]
+# we always need the xeus-python kernel for jupyterlite
+xeus-python = "*"
+python = "*"
+# add any extra packages from emscripten-forge here
+matplotlib = "*"
+
+[feature.host]
+channels = ["conda-forge"]
+platforms = ["osx-arm64", "osx-64", "linux-64", "win-64"]
+dependencies = { python = "*", jupyterlite-xeus = "*", jupyter_server = "*" }
+
+# make sure that we have the WASM environment installed before building the app
+[feature.host.tasks]
+setup_wasm = { cmd = "pixi install -e wasm", inputs = ["pixi.lock"] }
+
+# build the jupyterlite app
+[feature.host.tasks.build]
+cmd = "jupyter lite build --XeusAddon.prefix=.pixi/envs/wasm"
```

Looks like the `contents` CLI flag may be missing to pick up the content?

```suggestion
cmd = "jupyter lite build --XeusAddon.prefix=.pixi/envs/wasm --contents files"
```

---

I moved the file to the `./files` folder and it's picked up correctly. 

However, I had to add `jupyter_server` to the host environment to make things work :)

---

ah ok. Yes `jupyter_server` is indeed needed to index the contents.

Otherwise maybe you could keep just the `notebooks` folder and provide `--contents notebooks`.